radix-engine/src/engine/modules/module.rs

```diff
@@ -49,8 +55,10 @@ pub enum SysCallInput<'a> {
     },
 }
 
+#[derive(Debug)]
 pub enum SysCallOutput<'a> {
-    Invoke { rtn: String },
+    Invoke { rtn: &'a dyn Traceable },
+    ReadOwnedNodes,
```

Comment:
Is this being used anywhere? If not, would rather not add this, as most likely the `read_visible_nodes` syscall will get removed in the near future.

Reply:
I just added it to match the `pre_sys_call` call, happy to remove both in this PR; neither is used (other than for logger)